package.json

```diff
@@ -181,7 +181,6 @@
 		"source-map-loader": "0.2.4",
 		"sprintf-js": "1.1.1",
 		"style-loader": "1.0.0",
-		"stylelint-config-wordpress": "13.1.0",
```

Comment:
I could be wrong here. It works without since it's a dependency of the scripts package. But since we reference it in the local `.stylelintrc.json` we might want to leave it here?

Reply:
Restored in https://github.com/WordPress/gutenberg/pull/23114/commits/a196618ca94ba20fbf621558beebd6717031c9af to fix the build https://travis-ci.com/github/WordPress/gutenberg/jobs/347926249#L375.

Reply:
I think I know what happened. It's a usual issue. I fix it as follows:

- add the same dependency in the root `package.json` file
- run `npm i`
- when in `package-lock.json` is still `dependencies` section for one of the updated packages I remove this section manually, so in the lock file part included belowe, there should be no dependencies section, if it's there it means those deps will be installed in the local `node_modules` subfolder in the package:
 ```json
  "@wordpress/scripts": {
			"version": "file:packages/scripts",
			"dev": true,
			"requires": {
				"@svgr/webpack": "^5.2.0",
				"@wordpress/babel-preset-default": "file:packages/babel-preset-default",
				"@wordpress/dependency-extraction-webpack-plugin": "file:packages/dependency-extraction-webpack-plugin",
				"@wordpress/eslint-plugin": "file:packages/eslint-plugin",
				"@wordpress/jest-preset-default": "file:packages/jest-preset-default",
				"@wordpress/npm-package-json-lint-config": "file:packages/npm-package-json-lint-config",
				"@wordpress/postcss-plugins-preset": "file:packages/postcss-plugins-preset",
				"@wordpress/prettier-config": "file:packages/prettier-config",
				"babel-jest": "^25.3.0",
				"babel-loader": "^8.1.0",
				"chalk": "^4.0.0",
				"check-node-version": "^3.1.1",
				"cross-spawn": "^5.1.0",
				"css-loader": "^3.5.2",
				"dir-glob": "^3.0.1",
				"eslint": "^7.1.0",
				"eslint-plugin-markdown": "^1.0.2",
				"ignore-emit-webpack-plugin": "^2.0.2",
				"jest": "^25.3.0",
				"jest-puppeteer": "^4.4.0",
				"markdownlint": "^0.18.0",
				"markdownlint-cli": "^0.21.0",
				"mini-css-extract-plugin": "^0.9.0",
				"minimist": "^1.2.0",
				"node-sass": "^4.13.1",
				"npm-package-json-lint": "^5.0.0",
				"postcss-loader": "^3.0.0",
				"prettier": "npm:wp-prettier@2.0.5",
				"puppeteer": "npm:puppeteer-core@3.0.0",
				"read-pkg-up": "^1.0.1",
				"resolve-bin": "^0.4.0",
				"sass-loader": "^8.0.2",
				"source-map-loader": "^0.2.4",
				"stylelint": "^13.6.0",
				"stylelint-config-wordpress": "^17.0.0",
				"thread-loader": "^2.1.3",
				"url-loader": "^3.0.0",
				"webpack": "^4.42.0",
				"webpack-bundle-analyzer": "^3.6.1",
				"webpack-cli": "^3.3.11",
				"webpack-livereload-plugin": "^2.3.0"
			},
			"dependencies": {
				"postcss-value-parser": {
					"version": "4.1.0",
					"resolved": "https://registry.npmjs.org/postcss-value-parser/-/postcss-value-parser-4.1.0.tgz",
					"integrity": "sha512-97DXOFbQJhk71ne5/Mt6cOu6yxsSfM0QGQyl0L25Gca4yGWEGJaig7l7gbCX623VqTBNGLRLaVUCnNkcedlRSQ==",
					"dev": true
				},
				"stylelint-config-recommended": {
					"version": "3.0.0",
					"resolved": "https://registry.npmjs.org/stylelint-config-recommended/-/stylelint-config-recommended-3.0.0.tgz",
					"integrity": "sha512-F6yTRuc06xr1h5Qw/ykb2LuFynJ2IxkKfCMf+1xqPffkxh0S09Zc902XCffcsw/XMFq/OzQ1w54fLIDtmRNHnQ==",
					"dev": true
				},
				"stylelint-config-recommended-scss": {
					"version": "4.2.0",
					"resolved": "https://registry.npmjs.org/stylelint-config-recommended-scss/-/stylelint-config-recommended-scss-4.2.0.tgz",
					"integrity": "sha512-4bI5BYbabo/GCQ6LbRZx/ZlVkK65a1jivNNsD+ix/Lw0U3iAch+jQcvliGnnAX8SUPaZ0UqzNVNNAF3urswa7g==",
					"dev": true,
					"requires": {
						"stylelint-config-recommended": "^3.0.0"
					}
				},
				"stylelint-config-wordpress": {
					"version": "17.0.0",
					"resolved": "https://registry.npmjs.org/stylelint-config-wordpress/-/stylelint-config-wordpress-17.0.0.tgz",
					"integrity": "sha512-qUU2kVMd2ezIV9AzRdgietIfnavRRENt4180A1OMoVXIowRjjhohZgBiyVPV5EtNKo3GTO63l8g/QGNG27/h9g==",
					"dev": true,
					"requires": {
						"stylelint-config-recommended": "^3.0.0",
						"stylelint-config-recommended-scss": "^4.2.0",
						"stylelint-scss": "^3.17.2"
					}
				},
				"stylelint-scss": {
					"version": "3.17.2",
					"resolved": "https://registry.npmjs.org/stylelint-scss/-/stylelint-scss-3.17.2.tgz",
					"integrity": "sha512-e0dmxqsofy/HZj4urcGSJw4S6yHDJxiQdT20/1ciCsd5lomisa7YM4+Qtt1EG4hsqEG1dbEeF855tec1UyqcSA==",
					"dev": true,
					"requires": {
						"lodash": "^4.17.15",
						"postcss-media-query-parser": "^0.2.3",
						"postcss-resolve-nested-selector": "^0.1.1",
						"postcss-selector-parser": "^6.0.2",
						"postcss-value-parser": "^4.1.0"
					}
				}
			}
		},
  ```
- run `npm i` again
- remove the dependency in the root `package.json` file
- run `npm i` again

Reply:
Are you saying that `stylelint-config-wordpress` should be removed from the root package.json?

After fixing the build I personally decided that the current behaviour is correct due to the use of a custom ruleset based on `stylelint-config-wordpress`. In the unlikely event that @wordpress/scripts would start using something else the config would be broken.

This will probably change with #22777 so I think we should keep that as is and only update the versions.